gnovm/pkg/gnolang/machine.go

```diff
@@ -1799,18 +1824,29 @@ func (m *Machine) NumFrames() int {
 }
 
 func (m *Machine) LastFrame() *Frame {
-	return &m.Frames[len(m.Frames)-1]
+	return m.Frames[len(m.Frames)-1]
+}
+
+// MustLastCallFrame returns the last call frame with an offset of n. It panics if the frame is not found.
+func (m *Machine) MustLastCallFrame(n int) *Frame {
```

Comment:
Just curious, why would we favor this instead of `LastCallFrame` (asking because of the swap in this PR)?